src/components/cardComponent/Card.tsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { fetchProducts } from '../../redux/action/ProductAction';
+
+interface Product {
+  id: number;
+  name: string;
+  price: string;
+  picture_urls: string;
+  rating: any;
```

Comment:
rating is having any?

Reply:
It is not declared in our database ( We don't have it their)